app/i18n/en-us/feedback.php

```diff
@@ -0,0 +1,134 @@
+	),
+	'extensions' => array(
+		'already_enabled' => '%s is already enabled',
+		'cannot_remove' => '%s cannot be removed',
+		'disable' => array(
+			'ko' => '%s cannot be disabled. <a href="%s">Check FreshRSS logs</a> for details.',
+			'ok' => '%s is now disabled',
+		),
+		'enable' => array(
+			'ko' => '%s cannot be enabled. <a href="%s">Check FreshRSS logs</a> for details.',
+			'ok' => '%s is now enabled',
+		),
+		'no_access' => 'You have no access on %s',
+		'not_enabled' => '%s is not enabled',
+		'not_found' => '%s does not exist',
+		'removed' => '%s removed',
+	),
+	'import_export' => array(
+		'export_no_zip_extension' => 'The ZIP extension is not present on your server. Please try to export files one by one.',
+		'feeds_imported' => 'Your feeds have been imported and will now be updated',
+		'feeds_imported_with_errors' => 'Your feeds have been imported, but some errors occurred',
+		'file_cannot_be_uploaded' => 'File cannot be uploaded!',
+		'no_zip_extension' => 'The ZIP extension is not present on your server.',
+		'zip_error' => 'An error occurred during ZIP import.',
+	),
+	'profile' => array(
+		'error' => 'Your profile cannot be modified',
+		'updated' => 'Your profile has been modified',
+	),
+	'sub' => array(
+		'actualize' => 'Updating',
+		'articles' => array(
+			'marked_read' => 'The selected articles have been marked as read.',
+			'marked_unread' => 'The articles have been marked as unread.',
+		),
+		'category' => array(
+			'created' => 'Category, %s, has been created.',
+			'deleted' => 'Category has been deleted.',
+			'emptied' => 'Category has been emptied',
+			'error' => 'Category cannot be updated',
+			'name_exists' => 'Category name already exists.',
+			'no_id' => 'You must specify the id of the category.',
+			'no_name' => 'Category name cannot be empty.',
+			'not_delete_default' => 'You cannot delete the default category!',
+			'not_exist' => 'The category does not exist!',
+			'over_max' => 'You have reached your limit of categories (%d)',
+			'updated' => 'Category has been updated.',
+		),
+		'feed' => array(
+			'actualized' => '<em>%s</em> has been updated',
+			'actualizeds' => 'RSS feeds have been updated',
+			'added' => 'RSS feed <em>%s</em> has been added',
+			'already_subscribed' => 'You have already subscribed to <em>%s</em>',
+			'cache_cleared' => '<em>%s</em> cache has been cleared',
+			'deleted' => 'Feed has been deleted',
+			'error' => 'Feed cannot be updated',
+			'internal_problem' => 'The newsfeed could not be added. <a href="%s">Check FreshRSS logs</a> for details. You can try force adding by appending <code>#force_feed</code> to the URL.',
+			'invalid_url' => 'URL <em>%s</em> is invalid',
+			'n_actualized' => '%d feeds have been updated',
+			'n_entries_deleted' => '%d articles have been deleted',
+			'no_refresh' => 'There are no feeds to refresh',
+			'not_added' => '<em>%s</em> could not be added',
+			'not_found' => 'Feed cannot be found',
```

Comment:
I'd rather change the others to `can't` but it's true that this is the one that curiously stands out.

Reply:
There is a good mix of contractions/non-contractions, I guess it depends on how formal of a tone we are going for.  Cannot is used pretty consistently, so I'll leave for this PR.  Contractions could be revised in a future PR addressing inconsistencies.